CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+
+- Configure name and email in git:
+
+  ```
+  git config --global user.name "Your Name"
+  git config --global user.email "youremail@yourdomain.com"
+  ```
+
+- Fork this repo
+
+- In your fork, create a branch for your feature
+
+   ```git checkout -b add-contributing-section-to-readme```
+
+- Develop your feature and create a git commit:
+  - Add your changes to the commit
+
+     ```git add README.md```
+
+   - Sign off your commit using the -s, --signoff option. Write a good commit message (see [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/))
+
+
+     ```git commit -s -m "Add Contributing section to Readme"```
+
+- [Create a personal access token in Github](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) to be able to push your changes
+
+- Push your changes:  ```git push```
+
+- Send a PR to ods-ci using Github's web interface 
+
+- Test your PR in Jenkins using the rhods-ci-pr-test pipeline
```

Comment:
probably we can add [Internal Red Hat Only] to this header, External testing via local host and cluster is also possibe.